Includes/theme.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef NEVOLUTIONX_THEME_H
+#define NEVOLUTIONX_THEME_H
+
+#include <string>
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wreturn-type"
+#include "../3rdparty/json.hpp"
+#pragma clang diagnostic pop
+
+
+class Theme {
+public:
+  struct MenuTheme {
+    std::string font;
+    // TODO: Actually support this in Font.
```

Comment:
Minor nit: font not Font

Reply:
Pretty sure I was referring to the `Font` class, so I think the capitalization is correct. Making it "font" would be ambiguous with the member var just above.

Reply:
Also sounds good